packages/jest/README.md

```diff
@@ -110,18 +114,95 @@ SA11Y_AUTO=1 SA11Y_CLEANUP=1 jest
+    "location": null,
+    "status": "failed",
+    "title": "should throw error for inaccessible dom"
+  }
+]
+```
+
+With sa11y results processor:
+
+-   Original JSON test result (failure with embedded a11y error) is disabled
+
+```json
+"assertionResults": [
+{
+"ancestorTitles": [
+"integration test @sa11y/jest"
+],
+"failureMessages": [
+"A11yError: 1 Accessibility issues found\n * (link-name) Links must have discernible text: a\n\t- Help URL: https://dequeuniversity.com/rules/axe/4.1/link-name\n    at Function.checkAndThrow (packages/format/src/format.ts:67:19)\n    at automaticCheck (packages/jest/src/automatic.ts:54:19)\n    at Object.<anonymous> (packages/jest/src/automatic.ts:69:13)"
+],
+"fullName": "integration test @sa11y/jest should throw error for inaccessible dom",
+"location": null,
+"status": "disabled",
+"title": "should throw error for inaccessible dom"
+},
+]
+```
+
+-   Each unique a11y failure in a test module is extracted as a new test failure and added to a new test suite using a11y metadata as key. This could result in increase of total test count and suite count in the results JSON.
+
+```json
+"assertionResults": [
+  {
+    "ancestorTitles": [
+      "integration test @sa11y/jest",
+      "integration test @sa11y/jest should throw error for inaccessible dom"
+    ],
+    "failureMessages": [
+      "Accessibility issues found: Links must have discernible text\nCSS Selectors: a\nHTML element: <a href=\"#\"></a>\nHelp: https://dequeuniversity.com/rules/axe/4.1/link-name\nTests: \"integration test @sa11y/jest should throw error for inaccessible dom\"\nSummary: Fix all of the following:\n  Element is in tab order and does not have accessible text\n\nFix any of the following:\n  Element does not have text that is visible to screen readers\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty\n  Element has no title attribute"
+    ],
+    "fullName": "Links must have discernible text: a",
+    "location": null,
+    "status": "failed",
+    "title": "should throw error for inaccessible dom"
+  }
+],
+```
+
+#### Limitations
+
+Automatic checks currently has the following limitations.
+
+-   Automatic check is triggered regardless of the test status which would result in the original test failure if any getting overwritten by a11y failures if any from automatic checks ([#66](https://github.com/salesforce/sa11y/issues/66))
+-   Tests using the sa11y jest api would get tested twice with automatic checks - once as part of the sa11y API in the test and again as part of the automatic check
+    -   a11y issues from automatic checks would overwrite the a11y issues found by the API
+    -   If the sa11y API has been added to the test to check specific intermediate states of the DOM, enabling automatic checks could result in missed a11y issues
```

Comment:
I think this point is also covered by the next one below.

Reply:
Yes it is related, but there is also a fine difference - Enabling automatic checks without using the API vs while also using the API. Please check if the rewording now makes sense.

Reply:
I guess to me since you say the automatic check is only at the end of the test it was clear to me that checking intermediate states of the DOM, whether you called the API or not, would not be caught. I'm fine to keep it worded like this though if you think it's more clear.

Reply:
Thanks Trevor. I will try to revisit and reword/remove it in a subsequent PR.